incubator/jaeger/templates/_helpers.tpl

```diff
@@ -1,4 +1,16 @@
 {{/* vim: set filetype=mustache: */}}
+
+{{/*
+Return the appropriate apiVersion for cronjob APIs.
+*/}}
+{{- define "cronjob.apiVersion" -}}
+{{- if ge .Capabilities.KubeVersion.Minor "8" -}}
```

Comment:
This won't work once 1.10 is out. Also, this expects alpha features to be enabled for < 1.8. Better use `Capabilities.APIVersions.Has`.

Reply:
Done. I didn't know this was possible! I just learned something new! 🎉

Reply:
As for:
> Also, this expects alpha features to be enabled for...

This has been documented in the **Prerequisites** section of the `README.md` file and the spark cronjob is disabled by default.